src/database.cpp

```diff
@@ -1,10 +1,19 @@
 #include "database.hpp"
 
+Database::Database()
+{
+    createRTree = false;
```

Comment:
You can set this already in the initializer list like you do below for the second constructor